code/modules/projectiles/ammo_boxes.dm

```diff
@@ -352,6 +352,21 @@
 /obj/item/ammo_box/magazine/explosive/empty
 	empty = TRUE
 
+/obj/item/ammo_box/magazine/blank
```

Comment:
Magazines inside the box seem to not appear correctly when deployed.

Reply:
Fixed